gslib/commands/cp.py

```diff
@@ -97,8 +97,8 @@
     gsutil cp -r dir gs://my-bucket
 
   If you have a large number of files to transfer you might want to use the
-  gsutil -m option, to perform a parallel (multi-threaded/multi-processing)
-  copy:
+  top-level gsutil -m option (see "gsutil help command_opts"), to perform a
```

Comment:
This doesn't work; should be `gsutil help options`.

Reply:
@starsandskies

Reply:
Thanks Matt, I'll put the fix in!